osbs/core.py

```diff
@@ -895,6 +899,28 @@ def import_image(self, name, stream_import, tags=None):
 
         return True
 
+    def _check_import_image_response(self, import_response):
+        check_response(import_response)
+
+        failed_images = []
+        failed_images_server_error = False
+
+        for image in import_response.json()['status']['images']:
+            if image['status']['status'] == 'Success':
```

Comment:
Do we care about verifying that the status descriptions are for the specific set of tags we care about? Is that something the API ensures?

Reply:
[ImageStreamImportStatus](https://docs.openshift.com/container-platform/3.6/rest_api/openshift_v1.html#v1-imagestreamimportstatus) is only populated with the images that were actually requested to be imported based on the [ImageStreamImportSpec](https://docs.openshift.com/container-platform/3.6/rest_api/openshift_v1.html#v1-imagestreamimportspec).